README.md

```diff
@@ -0,0 +1,16 @@
+# **Stock Market**
+
+## Project summary
+In this project I created a system which simulates a stock market. The stock market composed of some companies. The system enables the user to insert and remove companies from the stock market and to make different actions.
+
+##  About the Classes
+Two classes created to implement the system idea:
+
+1. *Company* - This class includes the required methods and represents a specific company.
+
+2. *StockMarket* - This class represents the whole stock market which contains a list of companies.
+
+## Tests
+In order to check the correctness of all the functions in both classes, I did a tests for each function by using 'pytest' testing tool.
+
```

Comment:
Where are all the tests you run?
what commands did you use?

Reply:
I added an examples